all_datasets_report.py

```diff
@@ -176,7 +178,12 @@ def get_report_data(
             report_url = f'{DARWIN_API_URL}/api/reports/{team_id}/annotation?group_by=dataset%2Cuser&dataset_ids={ids_str}&granularity=day&format=csv&include=user.email%2Cdataset.name'
             response = requests.get(report_url, headers=headers)
             reader = csv.reader(response.text.splitlines())
-            report_data += list(reader)
+            if first_batch:
+                first_batch = False
+                report_data += list(reader)
+            else:
+                next(reader)
+                report_data += list(reader)
```

Comment:
The first row of every response is CSV headers, so this is to prevent multiple sets of headers appearing in different rows

We only want the first row of the CSV to be headers, so only take the headers if this is the first API call